File: Java/NaturalNumbers.java
import java.util.Scanner;
public class NaturalNumbers {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n;
        System.out.print("Enter a number: ");
        n = sc.nextInt();
        for(int i=1; i<=n; i++) {
            System.out.print(i + " ");
        }
        System.out.println();
        sc.close();
    }
}

File: CS/NaturalNumbers.cs
using System;
class NaturalNumbers
{
    static void Main()
    {
        int n;
        Console.Write("Enter a number: ");
        n = Int32.Parse(Console.ReadLine());

        for (int i = 1; i <= n; i++)
        {
            Console.Write(i + " ");
        }
        Console.WriteLine();
    }
}